_includes/navigation.html

```diff
@@ -1,23 +1,17 @@
 <header id="header" class="container-flex">
-  <!-- Logo -->
-  <img src="{{site.path}}/images/signalk.svg" alt="Signal K Logo"  height="100" width="100" />
-
-  <!-- Page Title -->
-  <h1><a href="{{site.path}}/" id="logo">Signal K</a></h1>
-
   <!-- Nav -->
-  <a class="{{page.index}}" href="{{site.path}}/index.html">Home</a>
-  <a class="{{page.overview}}" href="{{site.path}}/overview.html">Overview</a>
-  <a class="{{page.installation}}" href="{{site.path}}/installation.html">Installation</a>
-  <a class="{{page.documentation}}" href="{{site.path}}/specification/latest/">Documentation</a>
-  <a class="{{page.demo}}" href="{{site.path}}/demo.html">Application Demo</a>
-  <a class="{{page.team}}" href="{{site.path}}/team.html">Team</a>
+  <a class="{{page.index}}" href="{{site.path}}/index.html">HOME</a>
```

Comment:
Not a huge deal, but instead of capitalizing the text, consider a CSS rule to capitalize instead.